spec/01-unit/010-router_spec.lua

```diff
@@ -1344,7 +1304,7 @@ describe("Router", function()
 
         local router = assert(Router.new(use_case))
 
-        local _ngx = mock_ngx("GET", "/users/123/profile/hello/world", {})
+        local _ngx = mock_ngx("GET", "/users/123/profile/hello/world", { ["host"] = "domain.org" })
```

Comment:
Friendly nudge to point out you're breaking the 80-char limit all over the place. ;)

(And that is to say that I'd rather see the rule relaxed for sensible cases like this than the to have the PR changed for this trivial reason.)